tests/sentry/api/endpoints/test_relay_projectconfigs.py

```diff
@@ -190,57 +185,60 @@ def test_relays_dyamic_sampling(client, call_endpoint, default_project, dyn_samp
 def test_trusted_external_relays_should_not_be_able_to_request_full_configs(
     add_org_key, call_endpoint, no_internal_networks
 ):
-    result, status_code = call_endpoint(full_config=True)
+    result, status_code = call_endpoint()
     assert status_code == 403
 
 
 @django_db_all
-def test_when_not_sending_full_config_info_into_a_internal_relay_a_restricted_config_is_returned(
+def test_when_not_sending_full_config_info_into_a_internal_relay_the_full_config_is_returned(
     call_endpoint, default_project
 ):
-    result, status_code = call_endpoint(full_config=None)
+    result, status_code = call_endpoint()
 
     assert status_code < 400
 
     cfg = safe.get_path(result, "configs", str(default_project.id))
-    assert safe.get_path(cfg, "config", "filterSettings") is None
-    assert safe.get_path(cfg, "config", "groupingConfig") is None
+    assert safe.get_path(cfg, "config", "filterSettings") is not None
+    assert safe.get_path(cfg, "config", "groupingConfig") is not None
 
 
 @django_db_all
-def test_when_not_sending_full_config_info_into_an_external_relay_a_restricted_config_is_returned(
+def test_when_not_sending_full_config_info_into_an_external_relay_the_full_config_is_returned(
     call_endpoint, add_org_key, relay, default_project
 ):
     relay.is_internal = False
     relay.save()
 
-    result, status_code = call_endpoint(full_config=None)
+    result, status_code = call_endpoint()
 
     assert status_code < 400
 
     cfg = safe.get_path(result, "configs", str(default_project.id))
-    assert safe.get_path(cfg, "config", "filterSettings") is None
-    assert safe.get_path(cfg, "config", "groupingConfig") is None
+    assert safe.get_path(cfg, "config", "filterSettings") is not None
+    assert safe.get_path(cfg, "config", "groupingConfig") is not None
 
 
 @django_db_all
-def test_trusted_external_relays_should_receive_minimal_configs(
+def test_trusted_external_relays_should_receive_full_configs(
     relay, add_org_key, call_endpoint, default_project, default_projectkey
 ):
     relay.is_internal = False
     relay.save()
 
-    result, status_code = call_endpoint(full_config=False)
+    result, status_code = call_endpoint()
 
     assert status_code < 400
 
+    # Sweeping assertion that we do not have any snake_case in that config.
```

Comment:
Copied all new lines of this test from `test_internal_relays_should_receive_full_configs` and just modified `assert safe.get_path(cfg, "config", "trustedRelays") != []`.